core/src/banking_stage/consumer.rs

```diff
@@ -741,15 +742,14 @@ impl Consumer {
         error_counters: &mut TransactionErrorMetrics,
     ) -> Result<(), TransactionError> {
         let fee_payer = message.fee_payer();
-        let budget_limits =
-            process_compute_budget_instructions(message.program_instructions_iter())?.into();
-        let fee = bank.fee_structure().calculate_fee(
+        let budget_limits = FeeBudgetLimits::from(process_compute_budget_instructions(
```

Comment:
nit: maybe change `budget_limits` to `fee_budget_limits` as not to be confused with `compute_budget_limits`. Sorry for so many `budget`s and `limit`s.